ydb/core/viewer/viewer_ut.cpp

```diff
@@ -1831,4 +1832,105 @@ Y_UNIT_TEST_SUITE(Viewer) {
         auto resultSets = json["Databases"].GetArray();
         UNIT_ASSERT_EQUAL_C(1, resultSets.size(), response);
     }
+
+    static const ui32 ROWS_N = 15;
+    static const ui32 ROWS_LIMIT = 5;
+
+    TString PostExecuteScript(TKeepAliveHttpClient& httpClient, TString query) {
+        TStringStream requestBody;
+        requestBody
+            << "{ \"query\": \"" << query << "\","
+            << " \"database\": \"/Root/Test\","
+            << " \"syntax\": \"yql_v1\","
+            << " \"stats\": \"none\" }";
+        TStringStream responseStream;
+        TKeepAliveHttpClient::THeaders headers;
+        headers["Content-Type"] = "application/json";
+        headers["Authorization"] = "test_ydb_token";
+        const TKeepAliveHttpClient::THttpCode statusCode = httpClient.DoPost("/query/script/execute?timeout=600000", requestBody.Str(), &responseStream, headers);
+        const TString response = responseStream.ReadAll();
+        UNIT_ASSERT_EQUAL_C(statusCode, HTTP_OK, statusCode << ": " << response);
+        return response;
+    }
+
+    TString GetOperation(TKeepAliveHttpClient& httpClient, TString id) {
+        TStringStream requestBody;
+        TStringStream responseStream;
+        TKeepAliveHttpClient::THeaders headers;
+        headers["Content-Type"] = "application/json";
+        headers["Authorization"] = "test_ydb_token";
+        id = std::regex_replace(id.c_str(), std::regex("/"), "%2F");
+        const TKeepAliveHttpClient::THttpCode statusCode = httpClient.DoGet(TStringBuilder()
+                                                            << "/operation/get?timeout=600000&id=" << id
+                                                            << "&database=%2FRoot%2FTest", &responseStream, headers);
+        const TString response = responseStream.ReadAll();
+        UNIT_ASSERT_EQUAL_C(statusCode, HTTP_OK, statusCode << ": " << response);
+        return response;
+    }
+
+    TString GetFetchScript(TKeepAliveHttpClient& httpClient, TString id) {
+        TStringStream requestBody;
+        TStringStream responseStream;
+        TKeepAliveHttpClient::THeaders headers;
+        headers["Content-Type"] = "application/json";
+        headers["Authorization"] = "test_ydb_token";
+        id = std::regex_replace(id.c_str(), std::regex("/"), "%2F");
```

Comment:
...really? ;)

Reply:
The ID might look like ydb://scriptexec/9?id=b0483b31-c6799774-2cdd707e-86cc56b6, so it seems necessary to replace `/` with `%2F`. 
Is this the correct approach? :)

Reply:
> The ID might look like ydb://scriptexec/9?id=b0483b31-c6799774-2cdd707e-86cc56b6, so it seems necessary to replace `/` with `%2F`. Is this the correct approach? :)

yeah, but use regex for that is mega overkill. look for urlencode (or actualy it should work without encoding / at all)